.github/workflows/startup.yml

```diff
@@ -34,8 +34,10 @@ jobs:
         run: mvn install --log-file mvn.out.txt
 
       - name: Clone shogun-docker
-        run: git clone https://github.com/terrestris/shogun-docker
-        working-directory: ../
+        uses: actions/checkout@v2
+        with:
+          repository: terrestris/shogun-docker
+          ref: 4fdcbc51fa2d20bcfb0bcafafdfdece83c863819
```

Comment:
Will this ref be updated automatically / dependency bot?

Reply:
I'll try to configure it like that